run-build-functions.sh

```diff
@@ -132,32 +132,17 @@ restore_node_modules() {
 
 run_yarn() {
   yarn_version=$1
-  if [ -d $NETLIFY_CACHE_DIR/yarn ]
-  then
-    export PATH=$NETLIFY_CACHE_DIR/yarn/bin:$PATH
-  fi
   restore_home_cache ".yarn_cache" "yarn cache"
 
-  if [ $(which yarn) ] && [ "$(yarn --version)" != "$yarn_version" ]
+  if [ "$(yarn --version)" != "$yarn_version" ]
```

Comment:
What happens if yarn doesn't exist?

Reply:
yarn is always installed on the system

Reply:
Albeit we install `yarn` by default, we do it for the given default node version within build image:
- https://github.com/netlify/build-image/blob/650ad5b24bef4bd1740bac32c70bb791722993bd/Dockerfile#L291-L298

I believe that by installing or sourcing a different node version configured by the customer, yarn won't be present?